module/Company/src/Service/Company.php

```diff
@@ -3,26 +3,34 @@
 namespace Company\Service;
 
 use Application\Service\FileStorage;
-use Company\Form\EditCategory;
-use Company\Form\EditCompany;
-use Company\Form\EditJob;
-use Company\Form\EditLabel;
-use Company\Form\EditPackage;
-use Company\Mapper\BannerPackage;
-use Company\Mapper\Category;
-use Company\Mapper\FeaturedPackage;
-use Company\Mapper\Label;
-use Company\Mapper\LabelAssignment;
-use Company\Mapper\Package;
-use Company\Model\Job;
-use Company\Model\Job as JobModel;
-use Company\Model\JobCategory as CategoryModel;
-use Company\Model\JobLabel as LabelModel;
-use Company\Model\JobLabelAssignment;
+use Company\Form\{
+    EditCategory as EditCategoryForm,
+    EditCompany as EditCompanyForm,
+    EditJob as EditJobForm,
+    EditLabel as EditLabelForm,
+    EditPackage as EditPackageForm,
+};
+use Company\Mapper\{
+    BannerPackage as BannerPackageMapper,
+    Category as CategoryMapper,
+    FeaturedPackage as FeaturedPackageMapper,
+    Job as JobMapper,
+    Label as LabelMapper,
+    LabelAssignment as LabelAssignmentMapper,
+    Package as PackageMapper,
+};
+use Company\Model\{
```

Comment:
This also caused some invalid type hints in the PhpDoc. They are also reported by PhpStan